src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -3186,10 +3179,7 @@ private void ReinferMethodAndVisitArguments(BoundCall node, TypeWithState receiv
         /// </summary>
         private void ReportMaybeNullFromTypeParameterValueIfNeeded(BoundExpression expr, TypeWithState typeWithState, FlowAnalysisAnnotations annotations)
         {
-            if (!expr.IsSuppressed && !typeWithState.IsNotNull && RequiresSafetyWarningWhenNullIntroduced(typeWithState.Type, annotations))
-            {
-                ReportDiagnostic(ErrorCode.WRN_ExpressionMayIntroduceNullT, expr.Syntax, GetTypeAsDiagnosticArgument(expr.Type));
```

Comment:
>WRN_ExpressionMayIntroduceNullT [](start = 43, length = 31)

You removed uses of `ErrorCode.WRN_ExpressionMayIntroduceNullT` but not the enum member and the error message text.

Reply:
The resource and `ErrorCode` will be removed in master. Resource changes cannot made to 16.4 at this time.

---
In reply to: [341716084](https://github.com/dotnet/roslyn/pull/39589#discussion_r341716084) [](ancestors = 341716084)